hieradata/dev/wso2/wso2am/1.10.0/mesos/default.yaml

```diff
@@ -34,7 +34,8 @@ wso2::mgt_hostname: marathon-lb.marathon.mesos
 wso2::clustering:
   enabled: true
   membership_scheme: mesos
-#  mesos:
+  mesos:
+    marathon_applications: wso2am-gateway-manager,wso2am-gateway-worker
```

Comment:
Hi Imesh,

Thanks you for the explanation. My idea was to avoid duplication for configuration data so a new comer finds it easy to find and understand.